nss-day-cohort-45/tabloidmvc-bangles
# #20 Create Comments

## Changes Made:
1. Created necessary comment components (Repository, Interface, Controller, Model, View)
2. Added Comment button to Details.cshtml in the Post view that takes the user to the Comment Create view
3. Comments can be saved to the database; user is redirected to the Post's Details view after saving one (will be directed to a comments view in the future)

## How to Check:
1. Fetch gt-comments and run project
2. Navigate to Posts, select View, and select the Comment button towards the top of the post
3. Fill out the data fields and click the save button; verify that you are redirected to the post's Details view
4. Check the database to verify that your comment has been saved
